docs/reference/migration/migrate_6_0/settings.asciidoc

```diff
@@ -0,0 +1,10 @@
+[[breaking_60_settings_changes]]
+=== Settings changes
+
+==== Snapshot settings
+
+The setting `cluster.routing.allocation.snapshot.relocation_enabled` that allowed shards with running snapshots to be reallocated to
+different nodes has been removed. Enabling this setting could cause allocation issues if a shard got allocated off a node and then
```

Comment:
I would call it an "internal setting" to make it more obvious that this is not really much of a breaking change.